### PR TITLE
[CP-1825][BH-1660] Update while on onboarding screen

### PR DESCRIPTION
**Pull request by awulkiew**
Commit DB changes

**Your checklist for this pull request**

Make sure that this PR:
- [X] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/services/db/databases/migration/settings_bell/1/up.sql [hunk 1]
```diff
@@ -1,10 +1,14 @@
 -- Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 -- For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
+-- Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+-- For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
```
Comment (SP2FET) [inline products/BellHybrid/services/db/databases/migration/settings_bell/1/up.sql:6]: please remove the additional copyright info. 
@swichu91 do you have any clue why that happened?

Comment (swichu91) [reply products/BellHybrid/services/db/databases/migration/settings_bell/1/up.sql:6]: It seems like a minor issue in `db_migration.py` script. I will have a look at it.

File: products/BellHybrid/services/db/databases/migration/settings_bell/current/34a5f438_Move_gs_onboarding_done_to_global_scope/.meta [hunk 2]
```diff
@@ -1,6 +0,0 @@
-{
- "id": "34a5f438-a1fe-4818-af74-d084dc67ddf2",
- "date": "2023-03-02 12:44:53",
- "message": "Move gs_onboarding_done to global scope",
- "parent": 0
-}
```
Comment (swichu91) [inline products/BellHybrid/services/db/databases/migration/settings_bell/current/34a5f438_Move_gs_onboarding_done_to_global_scope/.meta:6]: There should be no `down.sql` and `up.sql` files too. As a matter of fact, the whole 'current' folder should be removed by the migration script. I've re-checked it locally and it works the way I described. Are you sure you didn't modify it by a hand?

Comment (awulkiew) [reply products/BellHybrid/services/db/databases/migration/settings_bell/current/34a5f438_Move_gs_onboarding_done_to_global_scope/.meta:6]: I didn't modify anything. I only executed `python3 tools/db_migration.py commit` from main directory after building BellHybrid and setting the envvar to the build directory (AFAIR as relative dir).

Comment (awulkiew) [reply products/BellHybrid/services/db/databases/migration/settings_bell/current/34a5f438_Move_gs_onboarding_done_to_global_scope/.meta:6]: I did it for more or less master branch so if there was some file that shouldn't be there it was there in the first place. AFAIU it was added in 950cf78aa9fe3f30d25d5048f0ec46bf5fe2eac0

Comment (awulkiew) [reply products/BellHybrid/services/db/databases/migration/settings_bell/current/34a5f438_Move_gs_onboarding_done_to_global_scope/.meta:6]: AFAIU now it's ok.

**Review by swichu91 — APPROVED** [review]

**Review by dawidwojtas-mudita — APPROVED** [review]

**Pull request merged**